src/setupProxy.js

```diff
@@ -0,0 +1,10 @@
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+module.exports = function (app) {
+  app.use("/api",
+    createProxyMiddleware({
+      target: process.env.REACT_APP_API_SERVER_URI,
+      changeOrigin: true,
+      pathRewrite: { "/api": "/" },
+    }));
+};
```

Comment:
이부분 인덴팅이 이상한 것 같습니다. 
```suggestion
module.exports = function (app) {
  app.use("/api",
    createProxyMiddleware({
      target: process.env.REACT_APP_API_SERVER_URI,
      changeOrigin: true,
      pathRewrite: { "/api": "/" },
  }));
};
```